inference-engine/src/legacy_api/CMakeLists.txt

```diff
@@ -41,7 +41,9 @@ target_compile_definitions(${TARGET_NAME}_obj PRIVATE IMPLEMENT_INFERENCE_ENGINE
 target_include_directories(${TARGET_NAME}_obj PRIVATE
     ${PUBLIC_HEADERS_DIR}
     ${CMAKE_CURRENT_SOURCE_DIR}/src
+    ${IE_MAIN_SOURCE_DIR}/src/snippets/include
```

Comment:
can be removed since it's included below as `$<TARGET_PROPERTY:inference_engine_snippets,INTERFACE_INCLUDE_DIRECTORIES>`

Reply:
Thanks! Done